### DISCUSSION TITLE
Upload internationally customized images

### DISCUSSION THREAD

**Discussion by TimoFedd** [amzn/selling-partner-api-models#3245] [Ideas]
Seller central UI gives us the option to upload internationally customized images for each marketplace. (https://sellercentral.amazon.de/imaging/upload/country)
Is there any way to do it with the API?

**Comment by sellerventory** [comment]
Hi Timo, did you found out how to do it?

**Comment by rugved1991** [comment]
Hi @TimoFedd,

Currently it is not possible to upload country specific images on a listing through the API. You will have to use the Seller Central portal for this activity. I have added this feedback to our feature request backlog and will inform the internal team that you are looking forward to this feature.
If you were to create a separate listing for each marketplace through the API, then you can provide marketplace specific images in the listing submission. But, until then you will have to use the Seller Central UI.

Best,
Rugved
Solutions Architect, SP-API

**Comment by Everbrent** [comment]
Hi @TimoFedd,

for us, too, the option of uploading country-specific images via API would be a feature that we would very much like to use.
At the moment we have to work with ZIP files, which causes a lot of traffic and is limited to 1GB.

Please put me on the list and inform me as soon as it is possible to upload country specific images via API.

Best regards,
Martin
IT Architect, Everbrent

**Comment by sellerventory** [comment]
Hello Everybody!

Do you know is there any update to this case from Amazon?

I thank you in advance
Arsen

**Comment by GWPOPP** [comment]
I would also like to know when this topic will be implemented in the API, as we want to update the images automatically. Through automation, we could prevent outdated images from not being replaced, which can unfortunately happen with manual processes.

I thank you in advance
Gerd

**Reply by ritikacd** [reply]
Hello @GWPOPP 

Thank you for reaching out requesting an enhancement to the Listings API for image localization. 

We are currently working on implementing this change, however, we do not have an estimated time of when the implementation will be complete. As soon as this occurs, we will post an announcement in the [Announcements Section](https://developer-docs.amazon.com/sp-api/changelog) of our documentation site.

Please note that if your required to have images that are specific only to particular countries, can upload through Seller Central UI. To remove outdated images, please reach out to Seller Central support. 

Thanks,
Ritika Chaudhary
Selling Partner API Developer Services

**Comment by MaxiAxi-Jorn** [comment]
@ritikacd, is there any update on this matter? Has been worked on for quite some time now.

**Reply by ritikacd** [reply]
Hi @MaxiAxi-Jorn
The feature is still not available through the APIs. We reached out to the backend team to learn more. I'll update the ticket once we have more information. 

Thanks,
Ritika

**Comment by dominicmahlingrelaxdays** [comment]
Any update on this matter?

**Reply by ritikacd** [reply]
Hi @dominicmahlingrelaxdays

The feature is still not available through the APIs. Could you share more about how this is impacting your workflows? 

Thank you,
Ritika

**Reply by dominicmahlingrelaxdays** [reply]
Hello @ritikacd,

We have fully automated the upload of new products to Amazon. Currently, we use the API to successfully upload the products with default/non-localized images. However, the inability to upload localized images via the API creates a significant bottleneck.

Because we cannot do this via the API, we have to manually add the localized images later by creating a zip file for that timeframe that we then manually upload to the seller central. This workaround causes several issues for our business:

- Delayed Time-to-Market: While the product goes live immediately, the localized images go live 5 to 14 days after the initial launch, depending on our Content Team's capacity.
- Reduced Revenue: Products perform better with localized images. Launching with only non-localized images directly hurts our conversion rates during the crucial early days of a product's lifecycle.
- Broken Automation: It breaks an otherwise fully automated pipeline, forcing a "clunky" manual step to finish a listing that consumes valuable team resources.

Adding API support for localized images would allow us to launch fully optimized listings from day one.  Is there any timeline for when this might be supported?

**Reply by ritikacd** [reply]
Hi @dominicmahlingrelaxdays, 

Thank you for sharing this valuable feedback! We don't have a timeline to share at the moment, but we'll pass this along to the team to help prioritize this request. 

Thank you,
Ritika
